state/compatibility_test.go

```diff
@@ -0,0 +1,225 @@
+package state_test
```

Comment:
What I am missing here is a test case that exercises the (production) changes you made to `LoadFinalizeBlockResponse`.
In other words, if I take this PR and only apply the testing changes (without the production changes) is any test failing?

Reply:
Fixed it.